src/app/admin/components/project-list/project-list.component.spec.ts

```diff
@@ -98,50 +98,60 @@ describe('ProjectListComponent', () => {
         mockModalService = TestBed.get(ModalService);
     });
 
-    it(`opens create project dialog when create button is clicked and navigates to a new created project`,
-        componentTest(() => ProjectListComponent, fixture => {
-            spyOn(fixture.componentInstance.router, 'navigate').and.returnValue(true);
-            fixture.debugElement.query(By.directive(Button)).nativeElement.click();
-            tick();
-            fixture.detectChanges();
-
-            expect(mockModalService.fromComponentSpy).toHaveBeenCalled();
-            mockModalService.confirmLastModal(mockProject({uuid: '__NEW_PROJECT_UUID__'}));
-
-            expect(fixture.componentInstance.router.navigate).toHaveBeenCalledWith(['/admin/projects', '__NEW_PROJECT_UUID__']);
-        })
+    it(
+        `opens create project dialog when create button is clicked and navigates to a new created project`,
+        componentTest(
+            () => ProjectListComponent,
+            fixture => {
+                spyOn(fixture.componentInstance.router, 'navigate').and.returnValue(true);
+                fixture.debugElement.query(By.directive(Button)).nativeElement.click();
+                tick();
+                fixture.detectChanges();
+
+                expect(mockModalService.fromComponentSpy).toHaveBeenCalled();
+                mockModalService.confirmLastModal(mockProject({ uuid: '__NEW_PROJECT_UUID__' }));
+
+                expect(fixture.componentInstance.router.navigate).toHaveBeenCalledWith([
+                    '/admin/projects',
+                    '__NEW_PROJECT_UUID__'
+                ]);
+            }
+        )
     );
 
-    it(`deletes a project when delete project is clicked`,
-        componentTest(() => ProjectListComponent, fixture => {
-
-            tick();
-            fixture.detectChanges();
+    it(
+        `deletes a project when delete project is clicked`,
```

Comment:
Revert the new line

Reply:
Done for project-list, schema-list and microschema-list.